pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -430,6 +486,9 @@ func setupVault(vaultPath string, deprecated bool) error {
 		if err := os.MkdirAll(vaultPath, 755); err != nil {
 			return err
 		}
+		if err := createVaultConfigFile(); err != nil {
+			return err
+		}
```

Comment:
Should we locate this function in symmetric for zfs one function, i.e. `createVault`?